src/sentry/issues/grouptype.py

```diff
@@ -53,6 +53,11 @@ def add(self, group_type: Type[GroupType]) -> None:
     def all(self) -> List[Type[GroupType]]:
         return list(self._registry.values())
 
+    def get_auto_resolve_enabled(self) -> List[Type[GroupType]]:
+        return list(
+            group_type for group_type in self._registry.values() if group_type.enable_auto_resolve
```

Comment:
Let's use `self.all()` instead - helps with readability and it makes sure the logic of 'get all types' is isolated to that method only.